### PR TITLE
Purview Sharing - Brief update to examples and changelog

### PR DESCRIPTION
**Pull request by hopewelchmicrosoft**
# Description

The initial 1.0.0b3 update did not contain mentions of the breaking change to the `orderby` parameter in all listing shares definitions. This PR fixes the CHANGELOG accordingly, as well as update the parameter in the README and sample files.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/purview/azure-purview-sharing/CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,6 @@
...
-- Updated broken link in samples README to point to Azure/azure-python-sdk instead of developer fork
-
-### Other Changes
-
-- Migrated test recordings to the azure-sdk-assets repo
-- Added more sanitizers to conftest.py
-
-
-## 1.0.0b3 (2023-06-29)
+## 1.0.0b3 (2023-07-19)
```
Comment (kristapratico) [inline sdk/purview/azure-purview-sharing/CHANGELOG.md:3]: Did this already release or does the date need to be updated to today?

Comment (hopewelchmicrosoft) [reply sdk/purview/azure-purview-sharing/CHANGELOG.md:3]: It's already been released. Should the date stay the same for this kind of update?

Comment (kristapratico) [reply sdk/purview/azure-purview-sharing/CHANGELOG.md:3]: Ah I see. Yes, let's keep it as-is then. The date should reflect the day it was released.

File: sdk/purview/azure-purview-sharing/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,5 @@
...
-
-- Updated broken link in samples README to point to Azure/azure-python-sdk instead of developer fork
-
-### Other Changes
-
-- Migrated test recordings to the azure-sdk-assets repo
-- Added more sanitizers to conftest.py
-
-
 ## 1.0.0b3 (2023-06-29)
```
Comment (kristapratico) [inline sdk/purview/azure-purview-sharing/CHANGELOG.md:3]: Just want to double-check, it looks like azure-purview-sharing released on 2023-07-18. Is this right? 

https://pypi.org/project/azure-purview-sharing/#history

Comment (hopewelchmicrosoft) [reply sdk/purview/azure-purview-sharing/CHANGELOG.md:3]: Oh, yep, that's right. Sorry about that, thank you for catching it!

File: sdk/purview/azure-purview-sharing/CHANGELOG.md [hunk 3]
```diff
@@ -21,15 +9,19 @@
 
 ### Breaking Changes
 
-- In _operations.py, under the list_attached definition, the `skip_token` parameter was removed list definitions in favor of placing it within `nextLink`. 
+- In _operations.py, the `skip_token` parameter was removed from list definitions in favor of placing it within `nextLink`.
```
Comment (kevinmichaelbowersox) [inline sdk/purview/azure-purview-sharing/CHANGELOG.md:12]: Remove the period here.

**Review by kevinmichaelbowersox — APPROVED** [review]

**Review by kristapratico — APPROVED** [review]

**Pull request merged**